bedrock/mozorg/redirects.py

```diff
@@ -401,10 +399,10 @@ def decider(request, **kwargs):
     ),
     # Bug 936362
     # only upper-case for XBL. /xbl is a namespace URL for the standard.
-    redirect(r"^XBL/?$", "https://developer.mozilla.org/docs/XBL"),
-    redirect(r"^RDF/?$", "https://developer.mozilla.org/docs/RDF", re_flags="i"),
-    # Bug 1332008
-    redirect(r"^protocol/?$", "https://blog.mozilla.org/opendesign/"),
+    redirect(r"^XBL/?$", "https://www.w3.org/TR/xbl/"),
+    redirect(r"^RDF/?$", "https://www.w3.org/RDF/", re_flags="i"),
+    # Bug 1332008, 1525853
+    redirect(r"^protocol/?$", "https://protocol.mozilla.org"),
```

Comment:
Originally /protocol was the vanity for "`moz://a`" Easter egg in URL but that has changed since to /mission so this now has no real use — redirecting it to protocol site instead…

Reply:
(added the bug ref with inproduct/search/protocol url update for posterity)